src/github_services.py

```diff
@@ -138,15 +149,21 @@ def __process_activity(
     pull_request: github_domain.PullRequest,
     event: Dict[str, Any]
 ) -> None:
-    """Process activity and updates assignee timestamps.
-    """
+    """Process activity and update the respective timestamp when the assignee was assigned."""
```

Comment:
What does "activity" refer to here? Please be more specific.

Reply:
I have changed the function name and the docstring. Please let me if that works.